src/resources/Pods/ContainersData.js

```diff
@@ -23,52 +25,70 @@ export default function ContainersData({ type, containers, statuses }) {
     return null;
   }
 
-  const ContainerComponent = ({ container, status }) => (
-    <UI5Panel
-      disableMargin
-      title={container.name}
-      headerActions={
-        <Button
-          aria-label={'view-logs-for-' + container.name}
-          onClick={() => {
-            setLayout({
-              midColumn: null,
-              endColumn: null,
-              layout: 'OneColumn',
-            });
-            navigate(
-              `${window.location.pathname.replace(
-                window.location.search,
-                '',
-              )}/containers/${container.name}`,
-            );
-          }}
-        >
-          {t('pods.buttons.view-logs')}
-        </Button>
-      }
-    >
-      <LayoutPanelRow
-        name={t('common.headers.status')}
-        value={<ContainerStatus status={status} />}
-      />
-      {container.image && (
-        <LayoutPanelRow name={t('pods.labels.image')} value={container.image} />
-      )}
-      {container.imagePullPolicy && (
+  const ContainerComponent = ({ container, status }) => {
+    const state =
+      status?.state?.running ||
+      status?.state?.waiting ||
+      status?.state?.terminated;
```

Comment:
is there a possibility, that state is not presented in YAML? if yes it will be nice to have last case set as `UNKNOWN` with type `none`

Reply:
This is only used to get `state?.startedAt`. If it's null we return '-'

```
  if (!timestamp) {
    return EMPTY_TEXT_PLACEHOLDER;
  }
```